wf-galleon/wildfly_channel_in_zips.adoc

```diff
@@ -0,0 +1,125 @@
+---
+categories:
+- provisioning
+---
+= [Experimental] Update WildFly download distributions using channels
+:author:            Jeff Mesnil
+:email:             jmesnil@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+[abstract]
+As an user, I can download WildFly from a zip archive and be able to update it to a more recent release.
+
+== Overview
+
+WildFly now publishes https://repo1.maven.org/maven2/org/wildfly/channels/[channels] that is a mechanism to keep a WildFly installation up to date with the latest releases.
+
+This works out of the box if users is provisioning WildFly (using the wildfly-maven-plugin or Prospero).
+However for users that are downloading WildFly from our traditional distributions (zip & tgz archives) they can not take advantage of this mechanism.
+
+If our archives would work out of the box with channels, it would significantely simplify updating an installation.
+
+The workflow would be as simple as:
+
+1. Download WildFly version x.y.z from our download page
+2. Use it as normal
+3. A micro release of WildFly x.y.(z+ 1) is made available
+4. Run a script in the existing WildFly installation to update if from x.y.z to x.y.(z+1)
+5. Reload the server to run your applications on WildFly x.y.(z+ 1)
+
+If there are multiple available updates for WildFly, the user must be able to decide which one they want to update to.
+This lets the user decide whether they want to update to the latest micro release or the next major release.
+
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-19221
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/WFLY-19130
+
+=== Stability Level
+
+* [X] Experimental
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+* mailto:{bstansbe@redhat.com}[{Brian Stansberry}]
+* mailto:{bspyrkos@redhat.com}[{Bartosz Spyrko-Smietanko}]
+
+=== Testing By
+
+* [X] Engineering
+
+=== Affected Projects or Components
+
+* WildFly
```

Comment:
Prospero would probably need changes as well if we want to enable user to choose the update to apply.